scripts/build-helpers/lib/common.sh

```
@@ -0,0 +1,315 @@
+require_cmd() {
+  command -v "$1" >/dev/null 2>&1 || die "Missing required command: $1"
+}
+
+require_subscription_env() {
+  [[ -n "${RH_ORG:-}" ]] || die "Set RH_ORG (Red Hat organization ID)."
+  [[ -n "${RH_ACTIVATION_KEY:-}" ]] || die "Set RH_ACTIVATION_KEY."
+}
+
+ensure_repo() {
+  [[ -f "${REPO_ROOT}/Makefile" ]] || die "REPO_ROOT does not look like the notebooks repo: ${REPO_ROOT}"
+  cd "${REPO_ROOT}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)